Blish HUD/GameServices/ArcDps/Models/CombatEvent.cs

```diff
@@ -5,11 +5,29 @@
     /// </summary>
     public class CombatEvent {
 
+        /// <summary>
+        /// The current combat event.
```

Comment:
Just a minor nitpick, I would call it `The event data` or similar, due to the delay, current feels out of touch to me.